make/migrations/postgresql/0120_2.9.0_schema.up.sql

```diff
@@ -1 +1,7 @@
 CREATE INDEX IF NOT EXISTS idx_task_extra_attrs_report_uuids ON task USING gin ((extra_attrs::jsonb->'report_uuids'));
+
+/* Set the vendor_id of IMAGE_SCAN to the artifact id instead of scanner id, which facilitates execution sweep */
```

Comment:
Could you please elaborate why changing the vendor_id of IMAGE_SCAN from scanner-id to artifact-id will facilitate the execution sweep? Thanks

Reply:
Currently, the unified sweep logic is cleaning the executions by `vendor_id`, an easy-to-understand example is replication, we set the `vendor_id` to replication policy id then we can retain fixed executions per policy, likewise for scanning, we want retain one execution for every artifact so need the `vendor_id` to be the artifact id.